src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -58,6 +58,11 @@ def handle_exception(ex):  # pylint: disable=too-many-return-statements
     from azure.cli.core.azlogging import CommandLoggerContext
     from azure.common import AzureException
     from azure.core.exceptions import AzureError
+    import traceback
+
+    logger.debug("azure.cli.core.util.handle_exception is called with an exception:")
+    # Print the traceback and exception message
+    logger.debug(traceback.format_exc())
```

Comment:
Can you make sure the trace is of `ex`? `ex` is not used.

Reply:
[`format_exc`](https://docs.python.org/3.8/library/traceback.html#traceback.format_exc) prints the last exception and when `handle_exception` is called, `sys.exc_info()` corresponds to `ex`.